src-gui/src/renderer/components/modal/introduction/slides/Slide01_GettingStarted.tsx

```diff
@@ -0,0 +1,23 @@
+import { Typography } from '@material-ui/core'
+import SlideTemplate from './SlideTemplate'
+import imagePath from '../../../../../assets/walletWithBitcoinAndMonero.png'
+
+export default function Slide01_GettingStarted(props: slideProps) {
+    return (
+        <SlideTemplate
+            title="Getting Started"
+            {...props}
+            imagePath={imagePath}
+        >
+            <Typography variant="subtitle1">
+                To make a Atomic Swap with Unstoppable swap you need a
+            </Typography>
+            <Typography>
+                <ul>
+                    <li><strong>Bitcoin Wallet</strong> with <strong>Funds</strong> for the swap</li>
+                    <li>Monero Wallet to generate a <strong>Monero Redeem Address</strong></li>
```

Comment:
nitpick: The second "Monero" is a bit redundant

`Monero Wallet to generate a <strong>Redeem Address</strong>`

Reply:
I'd change to "Monero wallet to receive the Monero".